### PR TITLE
Fixed Inventory Preview

### PR DESCRIPTION
**Pull request by Psychpsyo**
This makes the inventory preview of an imported gif be cropped to its first frame instead of showing the entire spritesheet.

### CODE REVIEW

**Comment by kawaiiamber** [comment]
LGTM - I'll test it when I get back home.

**Comment by Psychpsyo** [comment]
> LGTM - I'll test it when I get back home.

Are you back home yet?

**Comment by kawaiiamber** [comment]
> > LGTM - I'll test it when I get back home.
> 
> Are you back home yet?

I've been personally a little busy with a new job and school starting. Let me rephrase:
I'll pull this locally / test it / merge it maybe when I find free time. Life is just very busy at the moment, unfortunately. I'll see if I have time next weekend to pull and test the changes.
